corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/form.less

```diff
@@ -134,6 +134,18 @@
 
 }
 
+.case-tile-container {
+    position: sticky;
+    position: -webkit-sticky;
```

Comment:
Is this still necessary? It looks like Safari supports regular `position: sticky` since late 2019.